src/static/client/app/common.scss

```diff
@@ -1,108 +1,81 @@
-// common.scss
-
 * {
     box-sizing: border-box;
     margin: 0;
     padding: 0;
   }
   
   body,
-html {
-  height: 100%;
-  font-family: 'Poppins', sans-serif; 
-  background-color: #fff; 
-}
+  html {
+    height: 100%;
+    font-family: Arial, sans-serif; //default font family unless overwritten
+    background-color: #fff; 
+  }
   
-  // Common layout
-  .layout {
+  .display-flex-center {
     display: flex;
     justify-content: center;
     align-items: center;
     height: 100vh;
   }
   
   .wrapper {
-    width: 100%;
     max-width: 400px;
     background-color: #fff;
     border-radius: 10px;
     box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1), 0 1px 3px rgba(0, 0, 0, 0.08);
     padding: 30px;
+    width: 100%;
   }
   
   #formContent {
     display: flex;
     flex-direction: column;
-  }
-  
-  // Text
-  .h2 {
-    text-align: center;
-    margin-bottom: 24px;
-  }
-  
-  // Form Components
-  .form-group {
-    margin-bottom: 20px;
-  }
-  
-  .form-label {
-    display: block;
-    text-align: center;
-    font-size: 18px;
-    font-weight: 600;
-    color: #4a4a4a;
-    margin-bottom: 10px;
-  }
-  
-  .form-input {
-    width: 100%;
-    padding: 10px;
-    font-size: 14px;
-    border: 1px solid #e0e0e0;
-    border-radius: 5px;
-    outline: none;
-    transition: border-color 0.3s;
   
-    &:focus {
-      border-color: #007bff;
+    .form-label {
+      display: block;
```

Comment:
bootstrap has a form-label field you can use: https://getbootstrap.com/docs/4.0/components/forms/
unless you are trying to do something specific here, then you can leave this, but double check if what your trying to do doesn't already exist in bootstrap